Marlin/src/feature/pause.cpp

```diff
@@ -538,7 +548,7 @@ void wait_for_confirmation(const bool is_reload/*=false*/, const int8_t max_beep
       HOTEND_LOOP() thermalManager.heater_idle[e].start(nozzle_timeout);
       TERN_(HOST_PROMPT_SUPPORT, host_prompt_do(PROMPT_USER_CONTINUE, GET_TEXT(MSG_REHEATDONE), CONTINUE_STR));
       TERN_(EXTENSIBLE_UI, ExtUI::onUserConfirmRequired_P(GET_TEXT(MSG_REHEATDONE)));
-      wait_for_user = true;
+      IF_DISABLED(FILAMENT_CHANGE_FAST_RESUME, wait_for_user = true);
```

Comment:
Note that `wait_for_confirmation` is also used by `M125`, so `FILAMENT_CHANGE_FAST_RESUME` will also affect that command, not just filament change. Therefore a better name might be `PAUSE_REHEAT_FAST_RESUME`.

Reply:
Fine by me, I didn't think about that while naming it.